BaseTools/BinWrappers/PosixLike/posix_path_env.yaml

```diff
@@ -6,6 +6,6 @@
 ##
 {
   "id": "binwrappers",
-  "scope": "global-nix",
+  "scope": "global",

```

Comment:
@Javagedes - This change is required (i.e. macOS requires the PosixLike files too). But obviously it's not correct to enable them for Windows, as I've done here. It might be useful to have a global Posix (maybe `global-psx`?) scope which is set on both macOS and Linux, but not on Windows?